### ISSUE TITLE
Using Logic test instead of Application Test in Unit testing Documentation 

### ISSUE THREAD

**Issue by johnlinvc** [RestKit/RestKit#898]
The Unit Testing Documentation in wiki is mainly focused on Model layer , object mapping and validation. Consequently, It should use Logic Test instead of Application Test.

According To Apple's [Unit test Documentation](http://developer.apple.com/library/ios/#DOCUMENTATION/DeveloperTools/Conceptual/UnitTesting/01-Unit-Test_Overview/overview.html), Application test is mainly about testing controllers.

**tl;dr.** remove test host setting in Unit testing Document.
